.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: test
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.x]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: |
+         cd maxdiff
```

Comment:
I wonder why `test.py` is ran here, and then after pytest is installed later, it is ran again?

Reply:
Removed.

Reply:
Thanks!